dts/bindings/ipc/zephyr,ipc-openamp-static-vrings.yaml

```diff
@@ -28,3 +28,8 @@ properties:
   mbox-names:
     description: MBOX channel names (must be called "tx" and "rx")
     required: true
+
+  priority:
+    description: WQ priority for the instance
```

Comment:
Am I missing a wider context where this is documented? If not, this really should get some more description. What are the min/max values? Are higher values higher priority, or are lower values higher priority? etc.

Reply:
you are right, I'll extend the description.

This priority value is translated into the WQ priority thread managing the incoming data on the IPC instance. So it really is a priority value for the thread as usually defined in Zephyr, so like this https://docs.zephyrproject.org/latest/reference/kernel/threads/index.html#thread-priorities 

The problem is that (AFAICT) we cannot use negative values here when we want to assign to the WQ an high (cooperative) priority so the idea is to resort to something more fancy like the one proposed in https://github.com/zephyrproject-rtos/zephyr/pull/43806#issuecomment-1073759252

Reply:
OK, thanks for explaining!

If this is a zephyr priority, then it's not really hardware configuration and so ordinarily I would say it shouldn't o into devicetree. But I understand the problem here: you don't really have anywhere else to configure it.

Can we compromise and call this `zephyr,wq-priority` to make it super obvious what's going on? I know the compatible is already zephyr-specific, but I'm hoping you get why this is a bit unusual for DT.

Anyway, as far as negative numbers go, you can use negative numbers in devicetree with some restrictions. Please read https://elinux.org/Device_Tree_Mysteries#Signed_Property_Values for details. Do you think the 2's complement restriction is still workable for your use case? If so, it might be better to do it that way.

Reply:
> Can we compromise and call this zephyr,wq-priority to make it super obvious what's going on? I know the compatible is already zephyr-specific, but I'm hoping you get why this is a bit unusual for DT.

If you think this is a dumb idea, go ahead and say so; I won't be offended.

Reply:
not at all. Actually that makes even more sense because the priority is really referring to the thread priority that is indeed something zephyr specific.